### PR TITLE
fix memory leak on fork gc

### PR DESCRIPTION
**Pull request by MeirShpilraien**

### CODE REVIEW

File: src/redis_index.c [hunk 1]
```diff
@@ -205,6 +205,9 @@ RedisSearchCtx *NewSearchCtxDefault(RedisModuleCtx *ctx) {
 }
 
 void SearchCtx_Free(RedisSearchCtx *sctx) {
+  //  if (sctx->keyName) {
```
Comment (mnunberg) [inline src/redis_index.c]: ?

Comment (MeirShpilraien) [reply src/redis_index.c]: I will remove the comment, could not put it here cause it cause issues on other location (which probably needs it)

Comment (mnunberg) [reply src/redis_index.c]: ohhhh i see.. comment with TODO or something

Comment (MeirShpilraien) [reply src/redis_index.c]: not sure it worth doing cause on master we do not have this issue ..

**Comment by MeirShpilraien** [comment]
- [ ] cherrypick 9af97fb5c7c271ec57136eac1ce6c8428b0a2e7e into master

**Pull request merged**